refinery/ui/source/js/tool-launch/partials/input-control-inner-nav.html

```diff
@@ -0,0 +1,27 @@
+ <div>
+  <button
+    class="btn btn-default btn-xs outline"
+    ng-disabled="$ctrl.currentGroup[$ctrl.currentGroup.length-2] == 0"
```

Comment:
This is fixed in my current branch which addresses who currentGroup is being used.